tests/helpers/skipif.py

```diff
@@ -0,0 +1,72 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from distutils.version import LooseVersion
+from typing import Optional
+
+import pytest
+import torch
+from pkg_resources import get_distribution
+
+from pytorch_lightning.utilities import _TORCH_QUANTIZE_AVAILABLE
+
+
+def skipif_args(
+    min_gpus: int = 0,
+    min_torch: Optional[str] = None,
+    avail_quant: bool = False,
```

Comment:
```suggestion
    quant_available: bool = False,
```

maybe not shorten `avail`? I don't think it's intuitive that `avail == available` for non-english speakers

Reply:
done